tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/AspNetCore/MvcOptionsIntegration.cs

```diff
@@ -0,0 +1,42 @@
+// <copyright file="MvcOptionsIntegration.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+#if !NETFRAMEWORK
+using System.ComponentModel;
+using Datadog.Trace.AppSec;
+using Datadog.Trace.ClrProfiler.CallTarget;
+using Datadog.Trace.Configuration;
+
+namespace Datadog.Trace.ClrProfiler.AutoInstrumentation.AspNetCore;
+
+/// <summary>
+/// The ASP.NET Core middleware integration.
+/// </summary>
+[InstrumentMethod(
+    AssemblyName = "Microsoft.AspNetCore.Mvc.Core",
+    TypeName = TypeName,
+    MethodName = ".ctor",
+    ReturnTypeName = ClrNames.Void,
+    MinimumVersion = Major2,
+    MaximumVersion = Major7,
+    IntegrationName = nameof(IntegrationId.AspNetCore),
+    InstrumentationCategory = InstrumentationCategory.AppSec)]
+[Browsable(false)]
+[EditorBrowsable(EditorBrowsableState.Never)]
+public class MvcOptionsIntegration
+{
+    private const string Major2 = "2";
+    private const string Major7 = "7";
```

Comment:
IMO, we should inline these into the attributes, there's no value in having them as constants, they just make it harder to do version updates 😅 

And related - we will need to bump these when I merge the .NET 8 release (which I hope to do very soon)